runtime/doc/options.txt

```diff
@@ -3541,23 +3465,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 	The value is set to 1 when it is not -1 and setting the 'keymap'
 	option to a valid keymap name.
 
-						*'imstatusfunc'* *'imsf'*
-'imstatusfunc' 'imsf'	string (default "")
-			global
-			{only available when compiled with |+xim| and
-			|+GUI_GTK|}
-	This option specifies a function that is called to obtain the status
-	of Input Method.  It must return a positive number when IME is active.
-
-	Example: >
-		function ImStatusFunc()
-		  let is_active = ...do something
-		  return is_active ? 1 : 0
-		endfunction
-		set imstatusfunc=ImStatusFunc
-<
-	NOTE: This function is invoked very often.  Keep it fast.
-
```

Comment:
Maybe these four options should remain as stubs? (see https://github.com/neovim/neovim/pull/4009#discussion_r49685443)